### ISSUE TITLE
No empty line between paragraphs

### ISSUE THREAD

**Issue by jacodg** [frankframework/frank-doc#79]
See for example the following pages:

https://frankdoc.frankframework.org/#!/Pipes/EscapePipe
https://frankdoc.frankframework.org/#!/Listeners/JavaListener
https://frankdoc.frankframework.org/#!/Pipes/BytesOutputPipe

In the Java source code you can see that the Javadoc of the class is divided into two paragraphs by a \<p> or newline but in the browser the paragraphs are are not divided by an empty line

**Comment by nielsm5** [comment]
`<p>` tag in (nieuwe) console moet margin-top 10px krijgen.

**Comment by Matthbo** [comment]
Is this also relevant in the new design?

**Comment by jacodg** [comment]
Yes, in my opinion some newlines would be nice in the following:

![Image](https://github.com/user-attachments/assets/41f1d80e-c1d6-44e4-8e9f-fd10eb42cd03)

Like is is done by Javadoc which makes it more readable:

![Image](https://github.com/user-attachments/assets/c90abc6b-ad83-43c3-8981-190f418133b0)
